api/openapi-spec/swagger.json

```diff
@@ -6632,12 +6632,29 @@
      "patches": {
       "type": "array",
       "items": {
-       "$ref": "#/definitions/v1.Patch"
+       "$ref": "#/definitions/v1.CustomizeComponentsPatch"
       },
       "x-kubernetes-list-type": "atomic"
      }
     }
    },
+   "v1.CustomizeComponentsPatch": {
```

Comment:
Please revert this file, unless you did it by purpose of this PR.
Reason is there is a known issue of flakiness generating swagger.json

Reply:
This pr is meant to address all the current dev issues we're encountering. This fixes the flaky swagger generation logic... at least for now. A better fix is welcome in the future, this is a simply way to get unblocked though that doesn't impact anything reallly.